src/patient/PatientRecord.jsx

```diff
@@ -812,6 +823,31 @@ class PatientRecord {
             return p.reasons && p.reasons.some(r => r.value.entryId && r.value.entryId === conditionEntryId);
         });
     }
+    
+
+    getSurgeriesForCondition(condition) {
+        const allProceduresForCondition = this.getProceduresForCondition(condition);
+        return allProceduresForCondition.filter((cond) => {
+            // Looking for Surgery code - should be based on http://ncimeta.nci.nih.govs
+            return cond.code === "C0851238";
+        });
+    }
+
+    getSurgeriesPlannedForCondition(condition) {
+        const surgeriesForCondition = this.getSurgeriesForCondition(condition);
+        return surgeriesForCondition.filter((cond) => {
+            // Looking for active request status code - should be based on http://hl7.org/fhir/STU3/valueset-request-status.html
+            return cond.status === "active";
```

Comment:
I'm not sure status is adequate for determining planned surgeries. I think we should be checking the expected performance date to see if it's after today